### PR TITLE
Remove gender-specific greeting message

### PR DESCRIPTION
**Pull request by chee**
Hey! Great package, thank you!

This message might be startling and unwelcoming for the members of the Common Lisp community who aren't brothers.

Possible replacements:

```lisp
"Take this REPL, may it serve you well"
```
^ more closely matches the rhythm of the Beatles line it's a reference to

```lisp
,(format "%s" ,"Take this REPL, %s, may it serve you well")
```

^ using the name like the "start of a beautiful program" greeting

```lisp
"Take this REPL, fellow hacker, may it serve you well"
```

^ used in [cider](https://github.com/clojure-emacs/cider/blob/1ed5163433c991c00ea83fdd4447e8daf4aeccbe/cider-util.el#L661)

```lisp
"You'll always find me in the REPL at parties."
```

^A reference to a song by _different_ British musician :D

### CODE REVIEW

File: sly.el [hunk 1]
```diff
@@ -1482,7 +1482,6 @@ Fall back to `sly-init-using-slynk-loader' if ASDF fails."
     "Hack and be merry!"
     "Your hacking starts... NOW!"
     "May the source be with you!"
-    "Take this REPL, brother, and may it serve you well."
```
Comment (PuercoPop) [inline sly.el:1485]: Would replacing brother with the gender neutral sibling work?

"Take this REPL, sibling, and may it serve you well."

I have seen "my sibling in Christ" as a way to make an expression gender-neutral. Maybe something like  'paren-loving sibling' would work better? 

Anyway, removing the message is still an improvement over the current situation

**Comment by joaotavora** [comment]
Javier Olaechea ***@***.***> writes:

> @PuercoPop commented on this pull request.
>
> -------------------------------------------------------------------------------------------------------------------------
>
> In sly.el:
>
>> @@ -1482,7 +1482,6 @@ Fall back to `sly-init-using-slynk-loader' if ASDF fails."
>      "Hack and be merry!"
>      "Your hacking starts... NOW!"
>      "May the source be with you!"
> -    "Take this REPL, brother, and may it serve you well."
>
> Would replacing brother with the gender neutral sibling work?
>
> "Take this REPL, sibling, and may it serve you well."
>
> I have seen "my sibling in Christ" as a way to make an expression gender-neutral. Maybe something like 'paren-loving
> sibling' would work better?
>
> Anyway, removing the message is still an improvement over the current situation

Sorry, but no.  At the risk of stating the obvious, this phrase doesn't
really mean that Sly thinks the user is its brother.  It's not meant to
be accurate conversation starter, rather a reference to an idiomatic
expression taken from popular culture, for comedic effect.  It has the
same "unwelcoming" power to non-men as "merry" has to chronic
melancholics.

I'm happy to add "sister" though, that's also nice.

João

**Comment by chee** [comment]
> It has the same "unwelcoming" power to non-men as "merry" has to chronic melancholics.
It is addressed at the user. It says "you", and it calls you "brother". As a non-man I can tell you i did find it unwelcoming after I spent the morning setting up common lisp on my computer to be addressed "Take this REPL, brother" here in my own home by my own text editor. 

I'm not trying to be awkward or anything, i mean... i already overrode it in my own init.el, but i can tell you that it does come across as unwelcoming. I'm not the first person to raise this. it was raised on [Slime in 2016](https://github.com/slime/slime/pull/323) and [Cider in 2013](https://github.com/clojure-emacs/cider/pull/415) where it was [remedied](https://github.com/clojure-emacs/cider/commit/83137613db48e473d5a8b8b41efa507f1a084081) on the very same day.

It's your project, of course, so it's chill if you're not interested in this. I raised the PR in the hopes of being helpful but I understand if you'd prefer to close.

**Comment by joaotavora** [comment]
Again, I'm happy to add "Take this here REPL sister".   Do you happen to know or think of of some funny one-liners that reference different genders that are not man or woman or any type of sexual preference??  I'd be delighted to add them, as long as they are not vulgar or demeaning, are at least remotely well-humored and mix in some Lisp culture.  

I think that fighting the injustice you allude to must focus on promoting previously suppressed or undervalued culture, not on suppressing more culture.

I'm pretty "chill", but I'm also pretty serious about this and I'd hate to lose you as a contributor given this opportunity. You can even add ASCII drawings other than the typical SLY cat.  Make a queer-themed Sly with some funky drawings, for example: I'm down with that.

**Comment by joaotavora** [comment]
"You'll always find me in the REPL at parties." is pretty nice ;-)

**Comment by chee** [comment]
That's ok. Thanks for your perspective and for reading the PR, i'll go ahead and close it. :)

**Comment by joaotavora** [comment]
Sure, thanks for understanding. Hit me up if you want to add a rabbit theme ;-) ! Kinda tired of those cats anyway, let rabbits of all colours have their day in the sun.

**Comment by Kevinpgalligan** [comment]
They told you that it felt unwelcoming to them and your response was "no it didn't" -- perhaps we should give more weight to the experience of people who are actually affected by this? :-D

I think that software should be conservative in avoiding gender-specific language. Not everyone is going to be familiar with an obscure Beatles reference and it could send an unwelcome message to non-Brothers that they are in a Brothers-only space. Even though I'm a Brother myself (and a Beatles fan), I felt vaguely embarrassed by the unnecessary gendering in this message when I saw it. It's in the same category as early computer magazine covers that showed angry wives berating their husbands for bringing home a new computer -- and god, it makes me feel so embarrassed on behalf of the tech community.

If you leave in the message it will continue to make people feel vaguely embarrassed and unwelcome. It costs almost nothing to remove it. Maybe this will remain a hill that you want to die on, just thought I'd throw in my 2 cents.

**Comment by joaotavora** [comment]
> They told you that it felt unwelcoming to them and your response was "no it didn't" -- perhaps we should give more weight to the experience of people who are actually affected by this? :-D

What conversion are you reading? That was not my response.

I'm not going to die on any hill, I'm just not going to remove this for the reasons stated. Just as I'm not going to edit the recording of 'Revolution Number 9'. Just add a good quote for "sister" there.  In fact add, as many as you want, as long as they're good and vaguely Lisp-related. Then the probability of getting "brother" is much reduced.  Or add a nice ASCII LGBTQ theme instead of those cats, and I'll make it the default.  This will require creativity, I know, which is so much harder to do than censorship.  But who knows, maybe you'll find it more fulfilling than sanitizing culture.

And when you do that and I merge your PR, I'll be here to defend your work and tell any brothers that feel threatened by the overwhelming references to non-brothers in SLY to go complain somewhere else.

**Comment by Kevinpgalligan** [comment]
> It has the same "unwelcoming" power to non-men as "merry" has to chronic melancholics.

This is essentially saying "no it didn't" and denying the person's feeling of being unwelcome.

Anyway, it seems like this is a hill you want to die on, so I will not waste our time further :) I'm sure you will be remembered as a martyr in the Culture Wars for centuries to come.

**Comment by mdbergmann** [comment]
> > It has the same "unwelcoming" power to non-men as "merry" has to chronic melancholics.
> 
> This is essentially saying "no it didn't" and denying the person's feeling of being unwelcome.

How would you know?

Should I be feeling unwelcome when I read 'sister'?

I think this gender thing gets a bit over the top. Whole languages are getting changed (i.e. German) and a third gender being added, so that 'brother' or 'sister' isn't enough anymore.
As if we had no other problem on this planet.

**Comment by joaotavora** [comment]
Look, it's pretty silly to be taking a quote out of context, when the whole context is right here.

> I'm sure you will be remembered as a martyr in the Culture Wars for centuries to come.

Maybe, but I've given you the change to fill SLY will as many inclusive references to historically oppressed groups as you want, and you chickened out.  So at least I won't be remembered as a coward.

**Comment by joaotavora** [comment]
@mdbergmann I respect your stance, but I don't agree with it. Historically, "sisters" have been immensely opressed and invisible by comparison to brothers. So it makes sense to take action today, on this planet, for this real problem. I just happen to think that that action can't be censorship, but rather creativity.

**Comment by mdbergmann** [comment]
Historically, yes. Now women are presidents of countries. In the western worlds at least. Not in other parts of the world, though.

**Comment by joaotavora** [comment]
> Historically, yes. Now women are presidents of countries. In the western worlds at least. Not in other parts of the world, though.

For that reasoning since EUA has had a black president, racism doesn't exist anymore.
It's not that easy I'm afraid. It helps, but stuff lingers, especially when you're on the side of the historically oppressed, because that oppression has found its way into deep into tissue of all parts of everyday life.  And that includes things dear to you, like your mother tongue, I'm afraid, we have to admit.  I'm not for changing language artificially, but I'm not for blissfully ignoring the fact that language codifies the less nice parts of history either.

Now I sound the ze professor :nerd_face:   I hope I'm not mansplaining :-D it's just my opinion anyway.  I really do hope more women and sisters make SLY and many other things their own.  

Incidentally, I thought of "hack and be gay!" to poke fun at "hack and be merry!". Happy to add it, but I'm not gay myself so it makes little sense for me to do it.

**Comment by monnier** [comment]
>> Historically, yes.  Now women are presidents of countries.  In the
>> western worlds at least.  Not in other parts of the world, though.

[ I'm not at all sure the "western world" is statistically different in
  this respect.  I'd welcome pointers to evidence in support of that (or
  its opposite).  ]

> Incidentally, I thought of "hack and be gay!" to poke fun at "hack and be
> merry!".  Happy to add it, but I'm not gay myself so it makes little sense
> for me to do it.

I don't see any need to be homosexual to add it.  If it comes to that
just wait until a time you're gay (in the sense of "merry") to push it :-)

As for a neutral replacement for "brother", I suggest "stranger".

**Comment by joaotavora** [comment]
> I don't see any need to be homosexual to add it.  If it comes to that
> just wait until a time you're gay (in the sense of "merry") to push it :-)

Hehe, I'll push it then if you think it's minimally funny.  Is it? I'd hate 
to be laughing alone. I'll add "sister" and "stranger" for good measure 
and @chee's "You'll always find me"... if she doesn't object.

**Pull request closed without merging**